### DISCUSSION TITLE
enable tls1 (java.securty) in pom or vm option for quarkus

### DISCUSSION THREAD

**Discussion by Chibilim** [quarkusio/quarkus#25133] [Community]
Hello,

i am working on a project using an old version of msql  and the access use TLS1.0
but tls1.0 is obsolet.

On local i just have to change java.security : jdk.tls.disabledAlgorithms

and to deploy on a docker i could change this file on the image.

but i would like a better solution using the pom.xml or command line vm option.

i have this [post](https://stackoverflow.com/questions/30429713/how-do-you-set-a-java-security-policy-in-maven-pom-xml-coming-from-ant ) which seems to do what i want but i did not manage to make it works with quarkus.

thanks for your anwers

**Comment by maxandersen** [comment]
What command with quarkus is it that specifically fails for you ?

**Reply by Chibilim** [reply]
this is not a command. I use panache quarkus to get data from a ms sql database. And i get an error about tls 1.0.
I manage to fix it by deleting TLSv1 in 
`jdk.tls.disabledAlgorithms=SSLv3, TLSv1, TLSv1.1, RC4, DES, MD5withRSA, \
    DH keySize < 1024, EC keySize < 224, 3DES_EDE_CBC, anon, NULL`

My problem is more about configuration in order to deploy. In 'standart java/maven' i could specify a custom java.security which override the jdk one.
I would like to do the same for a quarkus project. I tried few things but without success

**Reply by maxandersen** [reply]
Yes. I get it. But which mvn command/phase are you specifically running ?

**Reply by Chibilim** [reply]
Hello, 

sorry but i am not sure of what is done behind a "quarkus dev" or a mvnw quarkus:dev , it will build and run my app (here for dev mode)

but to deploy i will have to build/package/etc.

i thinks the solution of my problem will be in the build.

**Reply by Chibilim** [reply]
```
...
      <plugin>
        <groupId>${quarkus.platform.group-id}</groupId>
        <artifactId>quarkus-maven-plugin</artifactId>
        <version>${quarkus.platform.version}</version>
        <extensions>true</extensions>
        <executions>
          <execution>
            <goals>
              <goal>build</goal>
              <goal>generate-code</goal>
              <goal>generate-code-tests</goal>
            </goals>
          </execution>
        </executions>
        <configuration>
          <source>${maven.compiler.source}</source>
          <target>${maven.compiler.target}</target>
          <compilerArgs>
          </compilerArgs>
          <jvmArgs>-Djava.security.properties="java.security"</jvmArgs>
        </configuration>
      </plugin>
...
```

i tried somethings like that but it does nohings :/
